### PR TITLE
#2489 fixed consumer mock=force not effected

### PR DESCRIPTION
**Pull request by jsen-joker**
## What is the purpose of the change

XXXXX

## Brief changelog

XXXXX

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by jsen-joker** [comment]
#2489 provider可以使用mock=force，但consumer无法使用force，只能用default或true。

**Comment by diecui1202** [comment]
@jsen-joker pls. check the ci failure.

```
[[1;34mINFO[m] There is 1 error reported by Checkstyle 8.9 with codestyle/checkstyle.xml ruleset.
[[1;31mERROR[m] src/main/java/org/apache/dubbo/config/ServiceConfig.java:[24,8] (imports) UnusedImports: Unused import - org.apache.dubbo.common.utils.ClassHelper.
```

File: dubbo-common/src/main/java/org/apache/dubbo/common/utils/ConfigUtils.java [hunk 1]
```diff
@@ -57,8 +57,9 @@ public static boolean isEmpty(String value) {
     }
 
     public static boolean isDefault(String value) {
-        return "true".equalsIgnoreCase(value)
-                || "default".equalsIgnoreCase(value);
+        return (value != null) && ("true".equalsIgnoreCase(value)
+                || "force".equalsIgnoreCase(value)
```
Comment (diecui1202) [inline dubbo-common/src/main/java/org/apache/dubbo/common/utils/ConfigUtils.java]: It's not suitable for adding 'force' as default value.

**Comment by diecui1202** [comment]
@zonghaishang could u pls. review this PR?

**Comment by zonghaishang** [comment]
I will take a look.

**Comment by carryxyh** [comment]
Close via https://github.com/apache/incubator-dubbo/pull/2742

**Pull request closed without merging**